doc/_static/mpl.css

```diff
@@ -1118,3 +1118,28 @@ div.viewcode-block:target {
 .sidebar-announcement p {
     margin: 0.4em 0.4em 0.6em 0.4em;
 }
+
+/* new main nav */
+nav.main-nav{
+    background-color: #002b47;
```

Comment:
Background color should be same as h1 color? #11557C

Reply:
I like that the color is somewhat darker than the logo and headings. It gives some additional separation. With the same color, logo nav and headings would blend too much into each other.
Might be a tiny bit lighter though (in between current and the headings color).

Reply:
Its already light on dark, so I bet that would stand out more than adequately.  I think limiting your color palette and fonts is something most web design tips will suggest...

Reply:
I did try making the nav the same color as the logo, and it made the page look very flat. Tim is correct that the darker nav creates visual separation between the elements. I don't recommend making the logo and the nav the same color. 

Limiting a designs color palette is usually a good plan. However, the site color scheme currently only has two colors (logo blue and orange) and, unfortunately, it's pretty much impossible to build a meaningful site hierarchy with only two colors. 

Many websites have formal guidelines about how to apply color (see https://material.io/design/color/ for Google, https://primer.style/css/utilities/colors for Github). It might be nice to re-think the site color palette using the colors in Viridis in the future, but this would be a much larger and more complex project.

Reply:
These things are purely aesthetic, so whatever folks decide works is fine.  Here is what I was suggesting (plus or minus) for comparison:

<img width="1229" alt="Front" src="https://user-images.githubusercontent.com/1562854/67454734-360fdf00-f5e0-11e9-8c3e-495e5624744b.png">